src/z_native.c

```diff
@@ -112,11 +112,19 @@ static void Z_RemoveBlock(memblock_t *block)
     }
     else
     {
+        if (block->prev->next != block)
+        {
+          I_Error("Z_RemoveBlock: Doubly-linked list corrupted!");
```

Comment:
Whitespace looks wrong. Check the coding style guidelines.

Reply:
The "native" implementation of the API seems to be used in mus2mid, midiread, <x>setup and <x>server from my reading of the CMakeLists.txt. I'll have a look at the coding style guidelines.

Reply:
Tried to fix up the indentation. Hopefully it looks better now